app/daemon.go

```diff
@@ -633,7 +633,7 @@ func (d *MenderShellDaemon) routeMessageStopShell(message *ws.ProtoMsg) error {
 			}
```

Comment:
Also, line 626 above will `return` with no calling `session.MenderShellDeleteById(s.GetId())` -> won't this trigger the same kind of regression?